client.go

```diff
@@ -63,7 +70,7 @@ func NewClient(locale, username, password string) Client {
 func (h *Client) Product(code string) (*Product, error) {
 	request := h.newRequest("GET", "/api/v0/product/%s.json", code)
 
-	resp, err := http.DefaultClient.Do(request)
+	resp, err := (&http.Client{Timeout: h.timeout}).Do(request)
```

Comment:
rather than (re)creating an `http.Client` anew for each and every call to `Product`, we could add an `http.Client` field to `Client`.

Reply:
yeah.. it's better, and re(create) on a timeout change. I'll update it